test/integration/service/loadbalancer_test.go

```diff
@@ -249,7 +249,7 @@ func Test_ServiceLoadBalancerEnableLoadBalancerClassThenUpdateLoadBalancerClass(
 	}
 }
 
-func newCloudProviderController(t *testing.T, client *clientset.Clientset) (*servicecontroller.Controller, *fakecloud.Cloud, informers.SharedInformerFactory) {
+func newServiceController(t *testing.T, client *clientset.Clientset) (*servicecontroller.Controller, *fakecloud.Cloud, informers.SharedInformerFactory) {
```

Comment:
can you also explain the motivation for the rename in the PR description?
this is technically a cloud provider controller even if a fake cloud is returned by the function.

/sig network

Reply:
Hi @neolit123 , because this is a service controller in cloud provider, so I think it is better to name it `ServiceController` instead of `CloudProviderController`. You can refer this line: https://github.com/kubernetes/kubernetes/blob/master/test/integration/service/loadbalancer_test.go#L29, thanks!

Reply:
there is also line 265 which has:
```
t.Fatalf("Error creating cloud provider controller: %v", err)
```

Reply:
^ +1 to rename the log error below

Reply:
Thanks! Updated.